### ISSUE TITLE
SAFDnet waymo dataset  `./dist_test.sh xxx.yaml 8 output/ckpt/xxx.pth`  return: data/waymo/compute_detection_metrics_main: not found

### ISSUE THREAD

**Issue by du666888999** [zhanggang001/HEDNet#19]
After training with the Waymo dataset, running the command `./dist_test.sh xxx.yaml 8 output/ckpt/xxx.pth` ends with the error message: `/bin/sh: 1: data/waymo/compute_detection_metrics_main: not found`.

**Comment by du666888999** [comment]
I manually processed the inference results of the validation set from my model and the ground truth of the validation set from Waymo. I successfully ran them through the Waymo evaluation tool.
![d0c5d280be0268ed2babfff1e34bb0d](https://github.com/zhanggang001/HEDNet/assets/44111515/3b47c7b0-baaf-4b53-976b-2d4f77a45a07)

**Comment by zhanggang001** [comment]
> I manually processed the inference results of the validation set from my model and the ground truth of the validation set from Waymo. I successfully ran them through the Waymo evaluation tool. ![d0c5d280be0268ed2babfff1e34bb0d](https://private-user-images.githubusercontent.com/44111515/339750808-3b47c7b0-baaf-4b53-976b-2d4f77a45a07.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3NzMyMDcsIm5iZiI6MTcxODc3MjkwNywicGF0aCI6Ii80NDExMTUxNS8zMzk3NTA4MDgtM2I0N2M3YjAtYmFhZi00YjUzLTk3NmItMmQ0Zjc3YTQ1YTA3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDA0NTUwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU4YzMxMWRjYzhkNzhjODk3NDE2YmRjYWExZDc1YmVkNDQxYmFlOTk5YzE4YmQzMjU1OWEyNjBhMjk0MzZjMmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5rknR4n5x9CiRKrwaeIRscQP4jYOCMRn_y_rI5AEqik)

You have done a good job! We will upload the executable file for evaluation later. The `data/waymo/compute_detection_metrics_main: not found.` is a faster evaluation method.

**Comment by zheng0819** [comment]
> ###
> [Jun 14, 2024](#issuecomment-2167831004)

Hello, I am having the same problem, how can I use waymo evaluation tools for performance evaluation? Thanks a lot!

**Comment by zhanggang001** [comment]
> > [Jun 14, 2024](#issuecomment-2167831004)
> 
> Hello, I am having the same problem, how can I use waymo evaluation tools for performance evaluation? Thanks a lot!

Download here
[compute_detection_metrics_main](https://cloud.tsinghua.edu.cn/f/df1c6c0b629b4929a357/?dl=1)
[gt.bin](https://cloud.tsinghua.edu.cn/f/637df33ef3a44fd09fb8/?dl=1)

**Comment by zheng0819** [comment]
> > > [Jun 14, 2024](#issuecomment-2167831004)
> > 
> > 
> > Hello, I am having the same problem, how can I use waymo evaluation tools for performance evaluation? Thanks a lot!
> 
> Download here [compute_detection_metrics_main](https://cloud.tsinghua.edu.cn/f/df1c6c0b629b4929a357/?dl=1) [gt.bin](https://cloud.tsinghua.edu.cn/f/637df33ef3a44fd09fb8/?dl=1)

Thanks for the reply! I place these two files in the folder 'data/waymo', but I got another error:'Command 'data/waymo/compute_detection_metrics_main /root/tf-logs/HEDNet-main/tools/scripts/output/eval/epoch_24/val/default/result.bin data/waymo/gt.bin' returned non-zero exit status 127.', I cannot solve this, can you help me? Thanks a lot!

**Comment by zhanggang001** [comment]
> > > > [Jun 14, 2024](#issuecomment-2167831004)
> > > 
> > > 
> > > Hello, I am having the same problem, how can I use waymo evaluation tools for performance evaluation? Thanks a lot!
> > 
> > 
> > Download here [compute_detection_metrics_main](https://cloud.tsinghua.edu.cn/f/df1c6c0b629b4929a357/?dl=1) [gt.bin](https://cloud.tsinghua.edu.cn/f/637df33ef3a44fd09fb8/?dl=1)
> 
> Thanks for the reply! I place these two files in the folder 'data/waymo', but I got another error:'Command 'data/waymo/compute_detection_metrics_main /root/tf-logs/HEDNet-main/tools/scripts/output/eval/epoch_24/val/default/result.bin data/waymo/gt.bin' returned non-zero exit status 127.', I cannot solve this, can you help me? Thanks a lot!

em, did you use a linux platform? Maybe, you can send me your WeChat via email for more details.

**Comment by zheng0819** [comment]
> > > > > [Jun 14, 2024](#issuecomment-2167831004)
> > > > 
> > > > 
> > > > Hello, I am having the same problem, how can I use waymo evaluation tools for performance evaluation? Thanks a lot!
> > > 
> > > 
> > > Download here [compute_detection_metrics_main](https://cloud.tsinghua.edu.cn/f/df1c6c0b629b4929a357/?dl=1) [gt.bin](https://cloud.tsinghua.edu.cn/f/637df33ef3a44fd09fb8/?dl=1)
> > 
> > 
> > Thanks for the reply! I place these two files in the folder 'data/waymo', but I got another error:'Command 'data/waymo/compute_detection_metrics_main /root/tf-logs/HEDNet-main/tools/scripts/output/eval/epoch_24/val/default/result.bin data/waymo/gt.bin' returned non-zero exit status 127.', I cannot solve this, can you help me? Thanks a lot!
> 
> em, did you use a linux platform? Maybe, you can send me your WeChat via email for more details.

Thanks a lot! I have send my WeChat via email.
